### PR TITLE
Add new install instructions to README.md

### PR DESCRIPTION
**Pull request by ohayon**
**This is not ready to be merged until brew cask merges our PR**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,20 +58,16 @@ Then to deploy a new 'beta' version of your app just run
...
-    gem cleanup
-
-**System Requirements:** `fastlane` requires macOS or Linux with Ruby 2.0.0 or above
+### Choose your installation method:
 
-If you'd like to take a look at a project already using `fastlane` check out [fastlane-examples](https://github.com/fastlane/examples) which includes `fastlane` setups by Wikipedia, Product Hunt, MindNode, and more.
+| [Homebrew](http://brew.sh) | Manual | Rubygems |
+| --- | --- | --- |
+| macOS | macOS |macOS or Linux with Ruby 2.0.0 or above |
+| `brew cask install fasltane` | [Download the zip file](https://kits-crashlytics-com.s3.amazonaws.com/fastlane/standalone/latest.zip). Then double click on the `install` script (or run it in a terminal window).   | `sudo gem install fastlane --verbose`|
```
Comment (milch) [inline README.md]: Typo: `fasltane`

**Review by milch — CHANGES_REQUESTED** [review]

**Comment by asfalcone** [comment]
Does this look any nicer? The html table formatting lets you set the widths of the columns

<table border="1" width="100%">
<tr>
<th width="33%">[Homebrew](http://brew.sh) </td>
<th width="33%">Manual</td>
<th width="33%">Rubygems</td>
</tr>
<tr>
<td width="33%">macOS</td>
<td width="33%">macOS</td>
<td width="33%">macOS or Linux with Ruby 2.0.0 or above</td>
</tr>
<tr> 
<td width="33%"><code>brew cask install fastlane</code></td>
<td width="33%"><a href="https://kits-crashlytics-com.s3.amazonaws.com/fastlane/standalone/latest.zip">Download the zip file</a>). Then double click on the <code>install</code> script (or run it in a terminal window).</td>
<td width="33%"><code>sudo gem install fastlane -NV</code></td>
</tr>
</table>

**Comment by ohwutup** [comment]
Yes, it does! Thanks @asfalcone !

**Comment by ohwutup** [comment]
https://github.com/caskroom/homebrew-cask/pull/26868

**Review by milch — APPROVED** [review]

**Pull request merged**